scripts/upgrade_admin_server.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# This script starts the upgrade of admin server to the version of system and/or
+# Cloud product.
+# The script assumes that the repositories are already correctly changed to point
+# to the new product versions. After the package upgrade is done, script initiates
+# a reboot of the machine.
+
+
+upgrade_admin_server()
+{
+    # we will need the dump for later migrating it into postgresql
+    pushd /opt/dell/crowbar_framework
+    bin/rake db:migrate
+    bin/rake db:dump
+    popd
+
+    # Upgrade the distribution non-interactively
+    zypper --releasever 12.2 ref -f
+    zypper --non-interactive dist-upgrade -l --recommends --replacefiles
```

Comment:
Good question. We may create admin-server-upgraded-failure or something like that instead of -ok. Question is, if we should skip the reboot - it might have happened after the os upgrade or before...